### ISSUE TITLE
6.1.1 games are 'freezing' and CPU usage is stuck - no log written

### ISSUE THREAD

**Issue by pamduncan** [StrangeLoopGames/EcoIssues#2873]
Since 6.1.1 we are seeing a number of servers that game play started lagging out very badly so that payers in game could do nothing.  Server was reporting a high CPU use of a number between 30-50% and stayed stuck on that number within 2-3 percent.  Players who were not logged in could not log in - either they get stuck at entering world or at about 8% loading world.  Servers are not writing a crash report or any exception logs.  Restarting the server will let people in but the same thing starts happening withing a few minutes.

Server opted to create fresh worlds in case it was an issue with a 6.03 world but within a short time on a fresh world it happened again.  It happened so soon that no debris or much building had taken place so I dont think its related to waste or debris.  I am posting this so the various admins can post world to it that are doing this

**Comment by MarjieAVolk** [comment]
Related to https://github.com/StrangeLoopGames/EcoIssues/issues/2731?

**Comment by pamduncan** [comment]
I found another world in this shape today - there was a crash 2 hours before the server froze... that crash report is included.  then about 2 hours later the game essentially froze.  Server would not restart.  When checked there was a .tmp file (usually written during the world save process but it was not getting converted to Game.eco).  No further report in the dump folder but windows even manager was having issues with Game.eco as a file in use situation.    Link to file that cause original crash, the Game and Archive.eco and the temp file    The server is shut down and the temp but the Game.eco will not load.  Just sits there using about 70% CPU and the console will not open and the game will not load.  There are no crash or exception logs being written.  I left it for 30 minutes and it did not load.  I removed Game.eco and renamed Archive.eco wondering if Game.eco had gotten corrupt in this process.  The game has now loaded and is running although it did take some time to load uphttps://www.dropbox.com/s/zzh88iw7ons21pe/BIgWorld.zip?dl=0

**Comment by pamduncan** [comment]
I know that there was a question as to this being related to debris.  That's possible in the second world that I posted about as it had been played on.  However the server that prompted the initial report was a fresh start - why I mentioned in the initial report that I did not think it could be related to debris or waste as the issue happened before anyone had any time to really do anything.  Both of these server continue to get in this 'hung' state at least once a day and no logs are being written
